Tests/SonarScanner.MSBuild.Tasks.UnitTest/GetAnalyzerSettingsTests.cs

```diff
@@ -451,38 +463,9 @@ public void ShouldMerge_OldServerVersion_ReturnsFalse(string language)
         [DataTestMethod]
         [DataRow("cs")]
         [DataRow("vbnet")]
-        public void ShouldMerge_Multiples_NewServer_NoSetting_ReturnsTrue(string language)
-        {
-            // ignoreIssues should default to false => we want to process Roslyn issues => should merge
-            var logger = CheckShouldMerge("7.4.0.0", language, ignoreExternalIssues: null /* not set */, expected: true);
-            logger.AssertDebugLogged($"sonar.{language}.roslyn.ignoreIssues=false");
-        }
-
-        [DataTestMethod]
-        [DataRow("cs")]
-        [DataRow("vbnet")]
-        public void ShouldMerge_NewServerVersion_SettingIsTrue_ReturnsFalse(string language)
-        {
-            var logger = CheckShouldMerge("7.4.0", language, ignoreExternalIssues: "true", expected: false);
-            logger.AssertDebugLogged($"sonar.{language}.roslyn.ignoreIssues=true");
-        }
-
-        [DataTestMethod]
-        [DataRow("cs")]
-        [DataRow("vbnet")]
-        public void ShouldMerge_NewServerVersion_SettingIsFalse_ReturnsTrue(string language)
-        {
-            var logger = CheckShouldMerge("7.4", language, ignoreExternalIssues: "false", expected: true);
-            logger.AssertDebugLogged($"sonar.{language}.roslyn.ignoreIssues=false");
-        }
-
-        [DataTestMethod]
-        [DataRow("cs")]
-        [DataRow("vbnet")]
-        public void ShouldMerge_NewServerVersion_InvalidSetting_NoError_ReturnsFalse(string language)
```

Comment:
why did you remove `ShouldMerge_NewServerVersion_InvalidSetting_NoError_ReturnsFalse`?

Reply:
It tested that a warning was logged when the `sonar.language.roslyn.ignoreIssues` property had an invalid value. As the property is not read anymore, nothing is logged so the test does not make any sense anymore.

Reply:
well, testing that nothing is logged / nothing blows up on an invalid setting is still a valid test :)